### DISCUSSION TITLE
nb plugin uninstall 的bug

### DISCUSSION THREAD

**Discussion by a2505** [nonebot/discussions#98] [想法]
当有1个以上的插件的时候，uninstall 插件会出现这种错误
![image](https://user-images.githubusercontent.com/69350942/152990965-1e6a7e7f-0afa-4750-9fb8-1edbf429c7b4.png)
再次启动加载pyproject.toml文件时会报错
![image](https://user-images.githubusercontent.com/69350942/152991318-4de33486-cd10-4807-af30-91fe8acfcf4b.png)

**Comment by yanyongyu** [comment]
plugins里多了个逗号

**Reply by a2505** [reply]
看出来了，提交一个bug

**Reply by yanyongyu** [reply]
这个逗号是nb-cli生成的？你有没有自己修改过toml文件？nb-cli使用的tomlkit操作文件，并没有直接对文件操作过

**Reply by a2505** [reply]
是nb-cli生成的

**Reply by a2505** [reply]
就是用nb-cli安装插件，然后卸载插件，就会多出一个逗号

**Reply by yanyongyu** [reply]
那可能是上游tomlkit的bug

**Reply by a2505** [reply]
嗯
